packages/api/src/external/hie/set-doc-query-progress-with-source.ts

```diff
@@ -11,44 +11,34 @@ import {
   setDocQueryProgress,
 } from "../../command/medical/patient/append-doc-query-progress";
 import { getPatientOrFail } from "../../command/medical/patient/get-patient";
-import { processDocQueryProgressWebhook } from "../../command/medical/document/process-doc-query-webhook";
 import { PatientDataCommonwell } from "../commonwell/patient-shared";
 import { PatientDataCarequality } from "../carequality/patient-shared";
 import { getCWData } from "../commonwell/patient";
 import { getCQData } from "../carequality/patient";
+import { handleWebhookBeingSent } from "../../command/medical/document/process-doc-query-webhook";
 
 export type HIEPatientData = PatientDataCommonwell | PatientDataCarequality;
 
+type StaticProgress = Pick<Progress, "status" | "total">;
+
 export type SetDocQueryProgressWithSource = {
   source: MedicalDataSource;
-} & SetDocQueryProgressBase &
-  (
-    | {
-        downloadProgress?: Progress | undefined | null;
-        convertProgress?: Progress | undefined | null;
-        reset?: false | undefined;
-      }
-    | {
-        downloadProgress?: never;
-        convertProgress?: never;
-        reset?: true;
-      }
-  );
+  downloadProgress?: StaticProgress | undefined | null;
+  convertProgress?: StaticProgress | undefined | null;
```

Comment:
Why 'null' in the middle of the code? We've been trying to keep this to DB and HTTP interfaces.

Reply:
Because we have [this logic](https://github.dev/metriport/metriport/blob/3d42ce11c40efd0c7a6cc1cc9228190b4d6ffe0c/packages/api/src/command/medical/patient/append-doc-query-progress.ts#L98-L115) where its null then we set it to undefined but i realized its not something used with the new logic ive created so ill remove